src/main/webapp/app/exercises/text/assess/text-assessment.service.ts

```diff
@@ -93,7 +93,7 @@ export class TextAssessmentService {
      * @param submissionId id of corresponding submission of type {number}
      */
     public cancelAssessment(exerciseId: number, submissionId: number): Observable<void> {
-        return this.http.put<void>(`${this.resourceUrl}/exercise/${exerciseId}/submission/${submissionId}/cancel-assessment`, null);
+        return this.http.post<void>(`${this.resourceUrl}/exercise/${exerciseId}/submission/${submissionId}/cancel-assessment`, null);
```

Comment:
found the changes :-) can you use undefined here instead of null? we would like to avoid null in the client